neqo-transport/src/tracking.rs

```diff
@@ -351,6 +353,23 @@ pub struct AckToken {
     ranges: Vec<PacketRange>,
 }
 
+/// The counts for different ECN marks.
+pub type EcnCount = EnumMap<IpTosEcn, u64>;
```

Comment:
You aren't 100% consistent about using this type.  Just check that you are using the alias for things like `EcnCount::default()` and so on.